src/Illuminate/Foundation/helpers.php

```diff
@@ -443,7 +443,7 @@ function encrypt($value)
     function has_errors($key = null, $bag = null)
     {
         $errors = session('errors');
-        if (if (is_null($errors) || ! $errors->any())) {
+        if (is_null($errors) || ! $errors->any()) {
```

Comment:
Even better, at top of code: `$bag = $bag ?: 'default'`

Then for the rest of the code you can do: `$errors->$bag->...`